### ISSUE TITLE
Unable to upgrade to graal ee, unknown distribution

### ISSUE THREAD

**Issue by spockz** [oracle/graal#3976]
**Describe the issue**
Using the documented command to upgrade my CE 21.3.0 install to EE fails with "unkown distribution" (including typo ;)).

**Steps to reproduce the issue**
Please include both build steps as well as run steps
1. ```
    $> gu upgrade --edition ee
    Error: Unkown GraalVM edition: ee
    ```

**Describe GraalVM and your environment:**
 - GraalVM version (latest snapshot builds can be found [here](https://github.com/graalvm/graalvm-ce-dev-builds/releases)), or commit id if built from source: **21.3.0-CE installed through SDKMan**
 - JDK major version: **11**
 - OS: **Fedora 34**
 - Architecture: **AMD64**

**Comment by Ondrej-Douda** [comment]
Thanks for finding this problem...
Fix will be available with next release...

For now you can use a workaround and include string `{ee=GraalVM Enterprise Edition}` in front of EE catalog, file `release` located in root folder of you GraalVM installation...
Last line in this file should look like this: `component_catalog="{ee=GraalVM Enterprise Edition}gds://oca.opensource.oracle.com/gds/meta-data.json|https://www.graalvm.org/component-catalog/v2/graal-updater-component-catalog-java11.properties"`...

**Comment by MartinBalin** [comment]
Fixed in 22.1.0
